package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@arcana/auth": "^0.1.3",
-    "@arcana/auth-core": "2.0.0-alpha.20",
+    "@arcana/auth-core": "^2.0.0-alpha.21",
```

Comment:
Remove the carat symbol

Reply:
[done](https://github.com/arcana-network/wallet-ui/pull/637/commits/0e1ee0c932337a49bb3a50566c0d2922cbfbf35f)